diff --git a/src/WORKSPACE.bazel b/src/WORKSPACE.bazel
index 4cc53489d..446d64d00 100644
--- a/src/WORKSPACE.bazel
+++ b/src/WORKSPACE.bazel
@@ -230,10 +230,12 @@ http_archive(
     name = "zip_code_ken_all",
     build_file_content = "exports_files([\"KEN_ALL.CSV\"])",
     url = "https://www.post.japanpost.jp/zipcode/dl/kogaki/zip/ken_all.zip",
+    sha256 = "0da4d242a0d3a2abd896b639dab286f0676490248195d9fb80b4817945d6ee1b",
 )
 
 http_archive(
     name = "zip_code_jigyosyo",
     build_file_content = "exports_files([\"JIGYOSYO.CSV\"])",
     url = "https://www.post.japanpost.jp/zipcode/dl/jigyosyo/zip/jigyosyo.zip",
+    sha256 = "4697485041e31016600044a77fe9ae7358e74ffdab06e934e996e253d02a109f",
 )
