diff --git a/src/config.bzl b/src/config.bzl
index 11bb88ea8..cdb2e6269 100644
--- a/src/config.bzl
+++ b/src/config.bzl
@@ -60,5 +60,5 @@ MACOS_QT_PATH = "/usr/local/Qt-6.5.2"
 ## SHA256 of zip code files.
 # When zip code files are prefetched and used via --repository_cache for offline build,
 # SHA256 values should be specified.
-SHA256_ZIP_CODE_KEN_ALL = None
-SHA256_ZIP_CODE_JIGYOSYO = None
+SHA256_ZIP_CODE_KEN_ALL = "85f13da74bf178ecb18083ec6d31e63bc4395c7817c3bb3587c7e86707609d2a"
+SHA256_ZIP_CODE_JIGYOSYO = "1ef5d43b5e5bf059117a2570fe95e0a9e3cf8a27f3e3882da283ad18460d2264"
