diff --git a/src/WORKSPACE.bazel b/src/WORKSPACE.bazel
index 4cc53489d..446d64d00 100644
--- a/src/WORKSPACE.bazel
+++ b/src/WORKSPACE.bazel
@@ -230,10 +230,12 @@ http_archive(
     name = "zip_code_ken_all",
     build_file_content = "exports_files([\"KEN_ALL.CSV\"])",
     url = "https://www.post.japanpost.jp/zipcode/dl/kogaki/zip/ken_all.zip",
+    sha256 = "d890bd17c3d95dd832f9c1cd97619c255df2e0cb066f14edbde30494192ab540",
 )
 
 http_archive(
     name = "zip_code_jigyosyo",
     build_file_content = "exports_files([\"JIGYOSYO.CSV\"])",
     url = "https://www.post.japanpost.jp/zipcode/dl/jigyosyo/zip/jigyosyo.zip",
+    sha256 = "4653044416ad81ca411ee77328ad61a12f51d81543e2bfeb886013299804dbb3",
 )
