diff --git a/src/config.bzl b/src/config.bzl
index 2675e04..7336fbe 100644
--- a/src/config.bzl
+++ b/src/config.bzl
@@ -71,5 +71,5 @@ MACOS_QT_PATH = "/usr/local/Qt-6.5.2"
 ## SHA256 of zip code files.
 # When zip code files are prefetched and used via --repository_cache for offline build,
 # SHA256 values should be specified.
-SHA256_ZIP_CODE_KEN_ALL = None
-SHA256_ZIP_CODE_JIGYOSYO = None
+SHA256_ZIP_CODE_KEN_ALL = "e5a32c0199ebc890ea5c48c06fd817a168b26d68ce1c3a86ba0ae1cdf6c2d1c4"
+SHA256_ZIP_CODE_JIGYOSYO = "5d52511ce6613fcc0c5b24ac59b463e927cff76ac466c2ae07a511a64bb5913f"
diff --git a/src/MODULE.bazel b/src/MODULE.bazel
index 577917e..ac6f1c9 100644
--- a/src/MODULE.bazel
+++ b/src/MODULE.bazel
@@ -261,7 +261,7 @@ http_archive(
 #
 # SHA256 as of 2024-08-14
 # SHA256_ZIP_CODE_KEN_ALL = "d2d177ef64b9459a618d8aaa96b6c5f081cb3f37f6e9c00ba912001e66b81f3e"
-SHA256_ZIP_CODE_KEN_ALL = None
+SHA256_ZIP_CODE_KEN_ALL = "e5a32c0199ebc890ea5c48c06fd817a168b26d68ce1c3a86ba0ae1cdf6c2d1c4"
 http_archive(
     name = "zip_code_ken_all",
     build_file_content = "exports_files([\"KEN_ALL.CSV\"])",
@@ -270,7 +270,7 @@ http_archive(
 )
 # SHA256 as of 2024-08-14
 # SHA256_ZIP_CODE_JIGYOSYO="ab6fd92df35e63c4566d29f70456da1603d8178ce3fec073f7c27b28d0af2e10"
-SHA256_ZIP_CODE_JIGYOSYO = None
+SHA256_ZIP_CODE_JIGYOSYO = "5d52511ce6613fcc0c5b24ac59b463e927cff76ac466c2ae07a511a64bb5913f"
 http_archive(
     name = "zip_code_jigyosyo",
     build_file_content = "exports_files([\"JIGYOSYO.CSV\"])",
