diff --git a/src/config.bzl b/src/config.bzl
index 11bb88ea8..cdb2e6269 100644
--- a/src/config.bzl
+++ b/src/config.bzl
@@ -60,5 +60,5 @@ MACOS_QT_PATH = "/usr/local/Qt-6.5.2"
 ## SHA256 of zip code files.
 # When zip code files are prefetched and used via --repository_cache for offline build,
 # SHA256 values should be specified.
-SHA256_ZIP_CODE_KEN_ALL = None
-SHA256_ZIP_CODE_JIGYOSYO = None
+SHA256_ZIP_CODE_KEN_ALL = "29788f5b893befa748de333551884596437f2fa4233c53e9121b56f6f7dff109"
+SHA256_ZIP_CODE_JIGYOSYO = "ef1505916652d7612a74ef94bafe93361f207c8a33ed4b8153c5eeb378d25514"
